index.html

```diff
@@ -0,0 +1,214 @@
+      rel="stylesheet"
+      media="screen and (min-width: 768px)"
+      href="css/desktop.css"
+    />
+    <!-- Mobile style file -->
+    <link
+      rel="stylesheet"
+      media="screen and (max-width: 768px)"
+      href="css/mobile.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="toggle-icon"></div>
+      <div class="menu">
+        <nav class="secondary-menu desktop-padding">
+          <ul>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-facebook.svg"
+                  alt="Facebook small icon"
+                />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="images/social/small-twitter.svg"
+                  alt="Twitter small icon"
+                />
+              </a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </nav>
+        <nav class="primary-menu desktop-padding">
+          <a href="/"><img src="images/logo.svg" alt="Logo" /></a>
+          <div class="primary-menu-content">
+            <ul>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Join</a></li>
+              <li><a href="#">Register</a></li>
+              <li><a href="#">News</a></li>
+            </ul>
+            <button type="button" class="register-menu-btn orange">
+              Register
+            </button>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <section id="hero" class="desktop-padding">
+        <div class="desktop-padding">
+          <h1 class="orange">"Hello Programmers!"</h1>
+          <div class="hero-title cocogoose">
+            Web development short term training
+          </div>
+          <p class="hero-description">
+            A short term web design and development training held at St. Mary's
+            University. A certificate for taking the training is prepared by the
+            university.
+          </p>
+          <h2 class="hero-date">2022.11.22(TUE)~14(THU)</h2>
+          <span class="hero-address"
+            >@ Twins Building, St. Mary's University</span
+          >
+        </div>
+      </section>
+      <section id="the-training">
+        <div class="training-section desktop-padding">
+          <h2 class="the-training-title the-underline">The Training</h2>
+          <div class="events">
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/lecture-talk.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Lecture</h2>
+              <p>Attend the lectures of the web development course training.</p>
+            </div>
+            <div class="event-item">
+              <img

```

Comment:
- [ ] This section is made up of 5 articles and as such they should each be wrapped in an `article` tag. kindly fix this

Reply:
- I think you should implement this change as suggested by @Munsa1 🌸